cmd/metricscollector/v1alpha3/file-metricscollector/main.go

```diff
@@ -58,10 +62,40 @@ var pollInterval = flag.Duration("p", common.DefaultPollInterval, "Poll interval
 var timeout = flag.Duration("timeout", common.DefaultTimeout, "Timeout to check if main process of worker container exit")
 var waitAll = flag.Bool("w", common.DefaultWaitAll, "Whether wait for all other main process of container exiting")
 
+func printMetricsFile(mFile string) {
+	for {
+		_, err := os.Stat(mFile)
+		if err == nil {
+			break
+		} else if os.IsNotExist(err) {
+			continue
+		} else {
+			klog.Fatalf("could not watch metrics file: %v", err)
+		}
+	}
+	cmd := exec.Command("tail", "-f", mFile)
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		klog.Fatalf("Fail to get stdout pipe of logging metrics file: %v", err)
+	}
+
+	cmd.Start()
+	reader := bufio.NewReader(stdout)
+	for {
+		if line, err := reader.ReadString('\n'); err != nil || io.EOF == err {
+			break
+		} else {
+			klog.Info(line)
+		}
+	}
+	cmd.Wait()
+}
+
```

Comment:
Should we use the third-party tool such as https://github.com/hpcloud/tail?

Reply:
cool, I can update it

Reply:
How do you signal the completion of tail logs?

Reply:
we doesn't need know the completion of tail logs, when the worker container exit, metrics collector main process will start to collect and parse the file, and then exit.

Reply:
Just wondering if there is a chance to miss certain worker container logs in this case.